### ISSUE TITLE
express-transform-bare-module-specifiers unpublished from npm?

### ISSUE THREAD

**Issue by neloe** [nodecg/express-transform-bare-module-specifiers#5]
When attempting to set up nodecg and running `npm install --production` as per the documentation, I get the following error:
```bash
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/express-transform-bare-module-specifiers/-/express-transform-bare-module-specifiers-1.0.2.tgz
npm ERR! 404
npm ERR! 404  'express-transform-bare-module-specifiers@1.0.2' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'nodecg'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Going to the [npm page](https://www.npmjs.com/package/express-transform-bare-module-specifiers) shows that it is no longer on npm.

Is there a plan to republish on npm, or will we need to install it in alternate ways to use nodecg?

Thanks!

**Comment by jimmy-collazos** [comment]
Package is keep unpublished in npm registry!
@MattMcNam @Hoishin @AlexVanCamp can you help?

thanks!

**Comment by Hoishin** [comment]
Sorry for the trouble. I'm working on it and should be fixed later today

**Comment by Hoishin** [comment]
new version published as 1.0.4

**Comment by jimmy-collazos** [comment]
@Hoishin thanks!
